sensor_kit_base_link2velodyne_top_base_link:
  x: 0.0
  y: 0.0
  z: 0.0
  roll: 0.0
  pitch: 0.0
  yaw: 0.00872665
sensor_kit_base_link2velodyne_left_base_link:
  x: 0.175064
  y: 0.60733
  z: -0.199681
  roll: -0.2021612
  pitch: 0.824772
  yaw: 1.304176    
sensor_kit_base_link2velodyne_right_base_link:
  x:  0.171889
  y: -0.579678
  z: -0.216495
  roll: 0.0384696
  pitch: 0.8573663
  yaw: -1.5500206
sensor_kit_base_link2gnss:
  x: -0.1
  y: 0.0
  z: -0.2
  roll: 0.0
  pitch: 0.0
  yaw: 0.0
sensor_kit_base_link2traffic_light_camera:
  x: 1.2810379227752366e-01
  y: -9.2835991418070245e-03
  z: -1.1765842063953642e-01 
  roll: -0.00789906
  pitch: -0.0614382
  yaw: 0.0408739
